packages/admin-api-client/README.md

```diff
@@ -0,0 +1,243 @@
+| getHeaders    | `(customHeaders?: {[key: string]: string}) => {[key: string]: string`                                                                                                       | Returns Admin API specific headers needed to interact with the API. If `customHeaders` is provided, the custom headers will be included in the returned headers object.                                                                                                                                                                                               |
+| getApiUrl     | `(apiVersion?: string) => string`                                                                                                                                          | Returns the shop specific API url. If an API version is provided, the returned URL will include the provided version, else the URL will include the API version set at client initialization.                                                                                                                                                                              |
+| fetch         | `<TData>(operation: string, options?:`[AdminAPIClientRequestOptions](#adminapiclientrequestoptions-properties)`) => Promise<Response>`                                          | Fetches data from Admin API using the provided GQL `operation` string and [AdminAPIClientRequestOptions](#adminapiclientrequestoptions-properties) object and returns the network response.                                                                                                                                                                                 |
+| request       | `<TData>(operation: string, options?:`[AdminAPIClientRequestOptions](#adminapiclientrequestoptions-properties)`) => Promise<`[ClientResponse\<TData\>](#clientresponsetdata)`>` | Requests data from Admin API using the provided GQL `operation` string and [AdminAPIClientRequestOptions](#adminapiclientrequestoptions-properties) object and returns a normalized response object.                                                                                                                                                                        |
+
+## `AdminAPIClientRequestOptions` properties
+
+| Name           | Type                     | Description                                          |
+| -------------- | ------------------------ | ---------------------------------------------------- |
+| variables?     | `Record<string, any>`    | Variable values needed in the graphQL operation      |
+| apiVersion?    | `string`                 | The Admin API version to use in the API request      |
+| customHeaders? | `{[key: string]: string}` | Customized headers to be included in the API request |
```

Comment:
I'm wondering about this name - should it be just `headers`? At this point these are the only headers the app is deliberately sending, not sure if we need to say they're custom here.

Reply:
We could definitely change this - it's actually defined in a common ApiClient level type so I can change it there and have all the API clients use the same interface.